frontend/src/js/store/usersSlice/thunks.ts

```diff
@@ -775,8 +775,8 @@ export const setTooltipReadState = createAsyncThunk(`${sliceName}/setTooltipRead
   return Promise.all(tasks);
 });
 
-export const setAllTooltipsReadState = createAsyncThunk(`${sliceName}/toggleHelptips`, (readState = READ_STATES.read, { dispatch }) => {
-  const updatedTips = Object.keys(HELPTOOLTIPS).reduce((accu, id) => ({ ...accu, [id]: { readState } }), {});
```

Comment:
Please, remove the `HELPTOOLTIPS` import so the lint check can pass :)

Reply:
🤦 👌